AOJ/03/03_C.py

```diff
@@ -0,0 +1,8 @@
+while True:
+    x,y=(int(a) for a in input().split())
+    if  x == 0 and y == 0: 
+      break
+    elif x <= y :
+          print(x,y)
+    else : 
+        print(y,x)
```

Comment:
1つ目の条件文でbreakを使っているので、条件文を分割しても問題ありません。条件文を分割することで可読性を上げることができます。
```suggestion
    if  x == 0 and y == 0: 
      break
    if x <= y :
          print(x,y)
    else : 
        print(y,x)
```